tests/unit/ui/components/TestUserAvatar.js

```diff
@@ -62,4 +61,12 @@ describe(__filename, () => {
     expect(root.find('.UserAvatar').find(Icon))
       .toHaveProp('name', 'anonymous-user');
   });
+
+  it('renders a preview image when supplied', () => {
+    const preview = 'https://example.org/image.jpg';
+    const root = renderUserAvatar({ preview });
+
+    expect(root.find('.UserAvatar-image')).toHaveLength(1);
+    expect(root.find('.UserAvatar-image')).toHaveProp('src', preview);
+  });
 });
```

Comment:
You added an `altText` prop to `UserAvatar` but it doesn't seem to be tested in here.